src/camera/camera3d.jl

```diff
@@ -418,6 +418,7 @@ function translate_cam!(scene, cam, t)
 
     cam.eyeposition[] = eyepos + trans
     cam.lookat[] = lookat + trans
+    update_cam!(scene, cam)
```

Comment:
These will run twice on interaction if you put them there - once in `on_pulse` or `on(mouseevent)` and once in these functions. I originally put them in `on_pulse` so I could merge rotations translations and zoom updates when they happen simultaneously. It's probably not a big deal to move these, but we should at least avoid duplicate calls.

Reply:
The problem is that if triggered programmatically using e.g. `translate_cam!` then `update_cam!` is not called.